fluent-syntax/src/parser.js

```diff
@@ -117,7 +115,8 @@ export default class FluentParser {
   getEntry(ps) {
     let comment;
 
-    if (ps.currentIs('/')) {
+    if (ps.currentIs('/') ||
+      (ps.currentIs('#') && (ps.peekCharIs('#') || ps.peekCharIs(' ')))) {
```

Comment:
Why does this peek one char ahead? Isn't a single `#` enough to branch into `getComment`?

Reply:
Unfortunately, not.
```
key = Value
#word
```

should not branch into getComment since the comment starting code is `"# "|"## "|"### "`.

Reply:
I feel like that should throw a `SyntaxError` inside of `getComment`.